test.sh

```diff
@@ -857,19 +885,26 @@ function run_btree_tests() {
 # Run remaining functionality-related tests from driver_test
 # ##################################################################
 function run_other_driver_tests() {
-    local use_shmem=$1
+    local use_shmem=""
+    if [ $# -eq 1 ]; then
+        use_shmem="$1"
+   fi
+
     local use_msg=
     if [ "$use_shmem" != "" ]; then
         use_msg=", using shared memory"
    fi
+    # shellcheck disable=SC2086
     run_with_timing "Cache test${use_msg}" \
-        "$BINDIR"/driver_test cache_test --seed "$SEED"
+        "$BINDIR"/driver_test cache_test --seed "$SEED" $use_shmem
```

Comment:
This batch of diffs is a bug-fix for existing tests ... we were inadvertently not running them for the `--use-shmem` case.